packages/api/src/types/index.ts

```diff
@@ -0,0 +1,15 @@
+export type NetworkError = {
```

Comment:
can we maybe move this error to `@formbricks/types/errors`?

Reply:
done, thanks for pointing this out